code/code/sys/client.cc

```diff
@@ -570,10 +570,9 @@ int Descriptor::read_client(char* str2) {
         return 0;
       }
 
-      sprintf(buf, "$n just rented with the %s client.", MUD_NAME_VERS);
-      act(buf, FALSE, character, NULL, NULL, TO_ROOM);
+      act(format("$n just rented with the %s client.") % MUD_NAME, false,
```

Comment:
I wonder if we still have anyone using that client.
Also, even if we rename the MUD, we're not really renaming the old Sneezy client, so I don't see a reason to change this line. As it's already done, though, there's also not much reason to revert this change :)

Reply:
Something had to be changed either way, since MUD_NAME_VERS was previously a `const char*` and is now an `sstring` after my changes. So I figured I might as well fix the sprintf usage even though MUD_NAME is still a `const char*`.

It would be nice to get rid of all that old Brutius client code. Though I know we recently had at least one player using it still, miraculously. He even sent me a working copy of the client to use in tracking down a crash bug.